### PR TITLE
Add external_fixed_ips compute field to vkcs_networking_router resource

### PR DESCRIPTION
**Pull request by morozovalekseywot**
- Add external_fixed_ips compute field to vkcs_networking_router resource. 
- Add example for sprut-neutron vpn tunnel.

### CODE REVIEW

**Review by ftersin — CHANGES_REQUESTED** [review]
the same attribute in router datasource is required as well

File: examples/vpnaas/sprut_to_neutron/neutron.tf [hunk 1]
```diff
@@ -0,0 +1,194 @@
+# Create neutron network
```
Comment (ftersin) [inline examples/vpnaas/sprut_to_neutron/neutron.tf:1]: i'd put the example to vnpaas root

Comment (morozovalekseywot) [reply examples/vpnaas/sprut_to_neutron/neutron.tf]: done

File: examples/vpnaas/sprut_to_neutron/vars.tf [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  type    = string
+  default = "aes-128"
+}
+
+variable "psk_key" {
+  type    = string
+  default = "any_psk_key"
+}
+
+data "vkcs_images_image" "image" {
```
[the remark this answers is not in the document]
Comment (morozovalekseywot) [reply examples/vpnaas/sprut_to_neutron/vars.tf]: move to base-image

File: examples/vpnaas/sprut_to_neutron/vars.tf [hunk 3]
```diff
@@ -0,0 +1,44 @@
+variable "sdn_sprut" {
+  type    = string
```
[the remark this answers is not in the document]
Comment (morozovalekseywot) [reply examples/vpnaas/sprut_to_neutron/vars.tf]: done

File: examples/vpnaas/sprut_to_neutron/neutron.tf [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+  name = "default"
+  sdn  = var.sdn_neutron
+}
+
+# Create vm in neutron net
+resource "vkcs_compute_instance" "vm_neutron" {
+  name              = "neutron-vm-tf-example"
+  availability_zone = "ME1"
+  flavor_name       = "STD3-1-2"
+  # key_pair        = <path_to_your_key_pair_file> # need for connect to vm by ssh
```
[the remark this answers is not in the document]
Comment (morozovalekseywot) [reply examples/vpnaas/sprut_to_neutron/neutron.tf]: I have fixed, you are right

**Review by ftersin — CHANGES_REQUESTED** [review]
lets move guide example into a separate pr or move it to a separate commit here
but in last case use --amend for fixes

File: examples/dc/interface/internet.tf [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Get external network with Internet access
```
Comment (ftersin) [inline examples/dc/interface/internet.tf:1]: rename the file to base-<smth>

Comment (morozovalekseywot) [reply examples/dc/interface/internet.tf:1]: base-internet-sprut and base-internet-neutron?

File: examples/dc/interface/main.tf [hunk 6]
```diff
@@ -1,7 +1,15 @@
...
+  description  = "tf-example-description"
   dc_router_id = vkcs_dc_router.dc_router.id
-  network_id = vkcs_networking_network.app.id
-  subnet_id = vkcs_networking_subnet.app.id
+  network_id   = vkcs_networking_network.app.id
+  subnet_id    = vkcs_networking_subnet.app.id
+}
+
+# Connect internet to the router
+resource "vkcs_dc_interface" "dc_interface_internet" {
```
Comment (ftersin) [inline examples/dc/interface/main.tf:11]: this should be a separate file including in its own section in example chapter of doc template

File: vkcs/networking/data_source_router.go [hunk 7]
```diff
@@ -90,6 +90,26 @@ func DataSourceNetworkingRouter() *schema.Resource {
...
+							Description: "IP address of the external gateway of a router.",
+						},
+						"subnet_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Subnet ID of the external gateway of a router.",
+						},
+					},
+				},
+				Description: "List of the external gateway of a router.",
```
Comment (ftersin) [inline vkcs/networking/data_source_router.go:111]: list of external gateways

File: vkcs/networking/resource_router.go [hunk 8]
```diff
@@ -112,6 +112,26 @@ func ResourceNetworkingRouter() *schema.Resource {
...
+							Description: "IP address of the external gateway of a router.",
+						},
+						"subnet_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Subnet ID of the external gateway of a router.",
+						},
+					},
+				},
+				Description: "List of the external gateway of a router.",
```
Comment (ftersin) [inline vkcs/networking/resource_router.go:132]: ditto

File: examples/dc/interface/internet.tf (hunk 5 above)
Comment (ftersin) [reply examples/dc/interface/internet.tf:1]: this is an example of dc router which is supported on sprut only

File: CHANGELOG.md [hunk 9]
```diff
@@ -7,6 +7,9 @@ description: |-
 
 # VKCS Provider's changelog
 
+#### v0.7.4 (unreleased)
+- Add external_fixed_ips computed field to vkcs_networking_router resource.
```
Comment (ftersin) [inline CHANGELOG.md:11]: and datasource

File: templates/dc/resources/vkcs_dc_interface.md.tmpl [hunk 10]
```diff
@@ -11,7 +11,10 @@ description: |-
 {{ .Description }}
 
 ## Example Usage
-{{tffile .ExampleFile}}
+{{tffile "examples/dc/interface/main.tf"}}
+
+## Connect dc router to internet
```
Comment (ftersin) [inline templates/dc/resources/vkcs_dc_interface.md.tmpl:16]: capitalize Internet

File: CHANGELOG.md (hunk 9 above)
Comment (morozovalekseywot) [reply CHANGELOG.md]: done

File: templates/dc/resources/vkcs_dc_interface.md.tmpl (hunk 10 above)
Comment (morozovalekseywot) [reply templates/dc/resources/vkcs_dc_interface.md.tmpl]: done

File: vkcs/networking/resource_router.go (hunk 8 above)
Comment (morozovalekseywot) [reply vkcs/networking/resource_router.go]: done

File: vkcs/networking/data_source_router.go (hunk 7 above)
Comment (morozovalekseywot) [reply vkcs/networking/data_source_router.go]: done

File: examples/dc/interface/main.tf (hunk 6 above)
Comment (morozovalekseywot) [reply examples/dc/interface/main.tf]: done

File: examples/dc/interface/internet.tf (hunk 5 above)
Comment (morozovalekseywot) [reply examples/dc/interface/internet.tf]: done

**Review by paaanic — APPROVED** [review]

**Review by ftersin — APPROVED** [review]

**Pull request merged**